x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/huggingface/HuggingFaceService.java

```diff
@@ -67,7 +67,7 @@ private static HuggingFaceEmbeddingsModel updateModelWithEmbeddingDetails(Huggin
             model.getServiceSettings().uri(),
             null, // Similarity measure is unknown
             embeddingSize,
-            null  // max input tokens is unknown
+            model.getTokenLimit()
```

Comment:
Double check me on this, we do want to grab the limit from the original model that it was parsed from right?